source/Cosmos.System2/LockSystem.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Cosmos.System
+{
+    public class LockSystem

```

Comment:
We already use a mutex class as a lock system, you should use it rather than creating another.
You can find the class here: https://github.com/CosmosOS/Cosmos/blob/impl-multitask/source/Cosmos.Core/Processing/Mutex.cs
And an example of use here: https://github.com/CosmosOS/Cosmos/blob/impl-multitask/source/Cosmos.Core/Memory/Old/Old_Heap.cs#L65

Reply:
thank you! ;)
working on that, will be ready pretty soon!

Reply:
Done! See [this commit](https://github.com/Nik300/Cosmos/commit/5184a701b839113c8173555d36fe65ca67cd2b90) to check if it's everything ok! ;)